src/test/java/io/cdap/plugin/sfmc/source/MarketingCloudRecordReaderTest.java

```diff
@@ -183,7 +148,12 @@ public void testFetchData() throws Exception {
     Mockito.doReturn(response).when(client).fetchObjectRecords(object, null);
     Mockito.doReturn(results).when(response).getObjects();
     marketingCloudRecordReader.initialize(split, null);
-    Assert.assertTrue(marketingCloudRecordReader.nextKeyValue());
+    try {
+      marketingCloudRecordReader.nextKeyValue();
+      collector.getOrThrowException();
+    } catch (IOException e) {
+      Assert.assertEquals(79, e.getMessage().length());
```

Comment:
Why are we comparing the exception message length here? Can we just compare the actual message?

Reply:
Done. Changed it to the actual message instead of length.